### PR TITLE
docs: updates contributing docs

### PR DESCRIPTION
**Pull request by lindseywild**
Adds information about the demo apps, where to find them, and some troubleshooting steps if local environments aren't loading.

### CODE REVIEW

File: docs/content/contributing.md [hunk 1]
```diff
@@ -61,22 +61,34 @@ By default, the system tests run in a headless Chrome browser. Prefix the test c
 
 ## Writing documentation
 
-Documentation is written as [YARD](https://yardoc.org/) comments directly in the source code, compiled into Markdown via `rake docs:build` and served by [Doctocat](https://github.com/primer/doctocat).
+Documentation is written as [YARD](https://yardoc.org/) comments directly in the source code, compiled into Markdown via `build exec rake docs:build` and served by [Doctocat](https://github.com/primer/doctocat).
 
 ### Storybook / Documentation / Demo Rails App
 
+* **Storybook**: Components can be interacted with, seen in different contexts, and controls can be maniuplated
```
Comment (camertron) [inline docs/content/contributing.md:68]:
```suggestion
* **Storybook**: Components can be interacted with, seen in different contexts, and controls can be manipulated
```

File: docs/content/contributing.md [hunk 2]
```diff
@@ -61,22 +61,34 @@ By default, the system tests run in a headless Chrome browser. Prefix the test c
...
+  * Typically runs on port 5000
+  * To rebuild stories, navigate to the `demo` directory (`cd demo`) and then run `bin/rails view_component_storybook:write_stories_json`
+* **Docs**: Generated YARD docs with examples, see components with usage instructions and examples
+  * Typically runs on port 5400
+  * To rebuild docs, run `bundle exec rake docs:build`
+* **Demo App**: See components on a plain page with no interfering framework or styling. Used to test functionality.
+  * Typically runs on port 4000 - visit `/rails/view_components/` in your browser
+  * To rerender the templates, you do not have to restart the server. Run this command and refresh the page.
```
Comment (camertron) [inline docs/content/contributing.md:76]: Which command does this bullet point refer to?

File: docs/content/contributing.md [hunk 3]
```diff
@@ -61,22 +61,34 @@ By default, the system tests run in a headless Chrome browser. Prefix the test c
...
-See the demo app at `/rails/view_components/` in your browser.
-To rerender the templates, you do not have to restart the server. Run this command and refresh the page.
+If you are running into issues or not seeing your updates, a few things you can try:
 
-```bash
-bundle exec rake docs:preview
-```
+* Delete the `overmind.sock` file and run `script/dev` again
+* Ensure you have run `script/setup`
+* Delete the `node_modules` file and rereun `script/setup`
```
Comment (camertron) [inline docs/content/contributing.md:89]:
```suggestion
* Delete the `node_modules` directory and re-run `script/setup`
```

**Review by camertron — APPROVED** [review]

File: docs/content/contributing.md (hunk 2 above)
Comment (lindseywild) [reply docs/content/contributing.md]: Whoops - thank you - updated! https://github.com/primer/view_components/pull/1016/commits/35f2be5867dc9ea7d5df7d8d46c9739e9c1ddb0e

**Pull request merged**